src/Traces.Common/Constants/ApaleoOneConstants.cs

```diff
@@ -8,6 +8,12 @@ public static class ApaleoOneConstants
 
         public const string PropertyLevelRelativeUrl = "/traces/property";
 
+        public const string LogoutRelativeUrl = "/logout";
```

Comment:
isn't this an absolute URL?

Reply:
depends on..
absolute could mean `https://...`, lets ask @randyrod

Reply:
I would say it is a relative url https://kb.iu.edu/d/abwp

Reply:
```suggestion
        public const string LogoutUrlAbsolutePath = "/logout";
```

if you want to be 100% correct 😉